rakelib/github_stats.rake

```diff
@@ -10,6 +10,9 @@ REPOS = %w[
 namespace :github_stats do
   desc 'Hit the Github API and grab data on open PRs'
   task get_open_prs: :environment do
+    # setup calculator for determining duration excluding weekends
+    schedule = { %i[mon tue wed thu fri] => [[0 * 3600, 24 * 3600]] }
```

Comment:
Is it worth including holidays?

Reply:
It does have the ability to include holidays and I did consider it but it is something that would have to be updated on a yearly basis. I figured the chance of that being overlooked in a year was probably fairly high vs. the small amount of variance that a few holidays being included in calculations might cause.